src/main/webapp/resources/js/apis/users/users.js

```diff
@@ -27,9 +27,9 @@ export const usersApi = createApi({
     Edit user details.
     */
     editUserDetails: build.mutation({
-      query: ({ userId, firstName, lastName, email, phoneNumber, role, locale, enabled }) => ({
+      query: ({ userId, firstName, lastName, email, phoneNumber, role, locale, oldPassword, newPassword, confirmNewPassword, enabled }) => ({
```

Comment:
I don't think we want to send these values in the body. Instead of body {param1, param2, ....} you should send them in params: {param1, param2, ...}

Reply:
I wasn't thinking. Please see 569b0c038019a7b8f8fb06cfa56490a68776629b. I've wondered if I should create a separate function for changing password.

Reply:
Sure that would be better in my opinion as well

Reply:
Please see 7001950c04e1febf362dcb3aacdb4ddaaaeb1250.